lib/index.js

```diff
@@ -103,7 +103,9 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       const {email, cursor = null, limit = 10} = req.query;
       const {actor} = req.user || {};
       const query = {'account.email': email};
-      if(cursor) {query['account.created'] = {$gte: cursor};}
+      if(cursor) {
+        query['account.created'] = {$gte: cursor};
```

Comment:
Should this be `meta.created`, I don't think `account.created` is a thing, and if that's true, then the sort needs to be updated as well. https://github.com/digitalbazaar/bedrock-account/blob/master/lib/index.js#L165

Are there any other fields that could be used to key off of for pagination?  I'm concerned about this method because I can imagine some bulk insert operation that results in multiple accounts having the same created timestamp which is only ms resolution.  If that happens, this system breaks down.

I heard mention of using a `.skip` operator on the query, wouldn't that be a more reliable method?  So instead of `cursor` the parameter would just be `skip` and then you do a query sorting by `foo` and skip N records instead of doing `created: {$gte: cursor}`.

Reply:
Also, what are the requirement around paging/sorting here?  Are we *only* going to be able to page accounts based on created date?  While that does seem useful, wouldn't one expect to be able to sort by other fields and page through that way?  Alphabetical by email address or something along those lines for instance.

Reply:
1. good catch and quite hard to believe my pagination test was not failing here. corrected it.
2. @dlongley also wanted pagination on account id and while I do think it is possible even with bulks writes created would be milliseconds apart I am going to refactor to sort on account id.
3. at the moment nothing in the spec is asking for sort features at all. pagination was just added. it would not be terribly hard to make sort work on different fields, but that feature was not requested in the current doc.
4. the only issue with sort based off out of account id is order date times support $gte however account id can be used with equals.

Reply:
p.s. do remember email is not unique so while sortable it is hard to paginate by it. mongo and did ids are probably best for pagination points.

Reply:
also seriously thanks for the meta.created can't believe I missed that.

Reply:
ok switched to account.id on both sort and paginate. will mark as resolved once I go through the code once again and make sure I'm not seeing any more glaring errors etc. p.s. pagination test works, but I might add one for limit too.